operation.go

```diff
@@ -155,15 +155,36 @@ func (operation *Operation) ParseParamComment(commentLine string, astFile *ast.F
 
 	//five possible parameter types.
 	switch paramType {
+	case "[]query":
```

Comment:
So "query" means the parameter should be sent over query. 
Now what "[]query" is ?